README.md

```diff
@@ -20,7 +20,7 @@ RSpec repos as well. Add the following to your `Gemfile`:
 
 ```ruby
 %w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
-  gem lib, :git => "git://github.com/rspec/#{lib}.git", :branch => 'master'
+  gem lib, git: "https://github.com/rspec/#{lib}.git", branch: 'master'
```

Comment:
The old style hash syntax needs to be maintained here, otherwise this is ok to merge.